code/modules/mob/mob_helpers.dm

```diff
@@ -508,4 +508,18 @@ It's fairly easy to fix if dealing with single letters but not so much with comp
 	. = ""
 
 	if(has_trait(TRAIT_DISSECTED))
-		. += "<span class='notice'>This body has been dissected and analyzed. It is no longer worth experimenting on.</span><br>"
\ No newline at end of file
+		. += "<span class='notice'>This body has been dissected and analyzed. It is no longer worth experimenting on.</span><br>"
+		
+/mob/living/proc/surgeries_examine()
+	. = ""
+	if(!surgeries.len)
+		return
+	for(var/datum/surgery/S in surgeries)
+		var/obj/item/bodypart/BP = S.operated_bodypart
+		var/zone = S.location
+		if(!get_location_accessible(src, zone))
+			continue
+		if(!BP)
+			. += "<span class='warning'>[src] has surgical markings where [p_their()] [parse_zone(zone)] should go!</span><br>"

```

Comment:
Shouldn't this src be He/She/It helper ? Most examine messages only state the name once.